docs/Wiki/faq.md

```diff
@@ -6,6 +6,17 @@ description: Frequently asked questions
 
 ## **If you have any issues please go to [the troubleshooting page.](installing-northstar/troubleshooting.md)**
 
+### Q: What is Northstar and how does it work? <a href="#what-is-northstar" id="what-is-northstar"></a>
+
+A: Northstar is a mod for Titanfall2 that allows you to run your own gameservers with custom gamemodes, weapons, maps, and skins.
+
+It consists out of the launcher that hooks into the game and scripts that replicate vanilla functionality and extend them.
+
+The launcher re-purposes the existing `server.dll` that is shipped  with the Titanfall2 client for running single player (sp is essentially just a local server) and then re-uses library to host servers that other players can join.
+From there a bunch of logic is added to ensure some levels of security as the vanilla client fully trusts server (which is fine for vanilla where servers are run by Respawn but is not enough for Northstar cause anyone can host a server including malicious entities) as well as additional features and functionality.
+
+The game scripts replicate vanilla functionality to mimic the gamemodes that can be played in vanilla as well as adding additional gamemodes for new custom gamemodes.
```

Comment:
> adding additional gamemodes for new custom gamemodes.

i think you rewrote this and forgot to remove the previous one

Reply:
Fixed via 75082fac5e7fabaa4996944b058044ffd2a84c33

Thanks ^^